src/obfuscation.ts

```diff
@@ -15,6 +15,38 @@ export function decodeBase64(input: string) {
   return base64.atobPolyfill(input);
 }
 
+export function obfuscatePrecomputedBandits(
+  salt: string,
+  bandits: Record<string, IPrecomputedBandit>,
+): Record<string, IPrecomputedBandit> {
+  const obfuscatedBandits: Record<string, IPrecomputedBandit> = {};
+
+  Object.entries(bandits).map((entry) => {
```

Comment:
Putting my functional programmer hat on: `map` is expected to produce an output. Since you are accumulating into a mutation variable `obfuscatedBandits` a for loop is more appropriate. 

```suggestion
  Object.entries(bandits).forEach((entry) => {
```

Reply:
good catch,